joystream-node.Dockerfile

```diff
@@ -17,7 +19,7 @@ RUN BUILD_DUMMY_WASM_BINARY=1 cargo clippy --release --all -- -D warnings && \
     cargo test --release --all && \
     cargo build --release
 
-FROM debian:buster
+FROM ubuntu:21.04
```

Comment:
Why did distro change? Just curious about this.

Reply:
Because the debian:buster image only has x86_64. ubuntu image has both x86_64 and arm64